gradoop-flink/src/main/java/org/gradoop/flink/io/api/DataSource.java

```diff
@@ -49,7 +51,7 @@
    *
    * @return temporal graph
    */
-  default TemporalGraph getTemporalGraph() throws IOException {
+  default TemporalGraph getTemporalGraph() {
```

Comment:
Should the throws clause really be removed? We can not make sure that an implementation of `DataSource` does not throw any exception. I also think it belongs there, because get functions for non-temporal graphs also declare an `IOException`.